docs/configuration.md

```diff
@@ -0,0 +1,146 @@
+---
+id: config
+title: Dapplet`s config
+---
+
+When creating a dapplet module, one of the important entities is the **dapplet's config**. It allows you to create and configure individual parameters for the module.
+
+For example: the network in which the dapplet will work, configure the monetization of the dapplet, change the url addresses of the server and the overlay part of the dapplet.
+
+The configuration is implemented through the `schema.json`.
+
+The schema follows the rules defined in http://json-schema.org/.
+The Dapplet settings UI is generated by [react-jsonschema-form](https://react-jsonschema-form.readthedocs.io/en/latest/usage/single/).
+
+In the production environment, we set up the **dapplet's config**, and specify the parameters that will be available in the browser through the extension.
+
+:::caution
+
+You can edit and create a **dapplet config** only from the module's working environment. User can use **dapplet config**, in extension.
+
+:::
+
+
+### Configuration
+
+The **dapplet's config** is located in the */config* folder.
+
+In the `schema.json` file, specify the dapplet settings that will be available to the user.
+
+The scheme consists of parameters:
+
+  ```js
+ {
+// schema type: always object
+  "type": "object",
+// the fields you want to specify in your schema. In this field you list only the names of the required parameters. There may be other parameters that are optional.
+  "required": [
+   ...
+  ],
+// field parameters
+   "properties": {
+    // parameter's name
+    "network": {
+        // field's type: string or number
+      "type": ...,
+        // the title of the field that the user sees
+      "title": ...,
+        // optional field visibility parameter. Boolean's value. Make the setting hidden for users. It's displayed in the developer and testing mode, not in the public.
+      "hidden": ...,
+        // optional field in select format. it sets the dropdown form for the setting. You should list the available values here. 
+      "enum": [
+        ...
+      ]
+    },
+    }
+}
+  ```
+
+![img](/img/con_01.png)
+
+In the *"properties"* we specify the parameters with their properties for each field, also in the form of objects.
+
+![img](/img/con_02.png)
+
+  Fields in the format for entering numeric values can have a maximum and minimum value limit and a step.
+
+  ![img](/img/con_05.png)
```

Comment:
Why do you show this picture here?
Don't use pictures of the code if you can cteate code block with this data